rightmesh/contracts/MeshToken.sol

```diff
@@ -30,6 +38,11 @@ contract MeshToken is CappedToken, PausableToken {
   onlyOwner
   returns (bool)
   {
+    // don't allow owner to change this for themselves
+    // otherwise whenNotPaused will not work as expected for owner,
+    // therefore prohibiting them from calling pause/unpause.
+    require(_address != owner);
```

Comment:
does this mean that *any* address other than the owner can updateAllowedTransfers? shouldn there be some restrictions on who can do this?